ts/features/itwallet/common/components/ItwBadge.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import { Image, StyleSheet, View } from "react-native";
+import {
+  Canvas,
+  LinearGradient,
+  RoundedRect,
+  vec
+} from "@shopify/react-native-skia";
+import { H5, IOColors } from "@pagopa/io-app-design-system";
+import I18n from "../../../../i18n.ts";
+
+/**
+ * This component renders the logo of the IT Wallet, image and text.
+ */
+// TODO: replace with the correct image [SIW-2412]
+export const ItwBadge = () => (
+  <View style={styles.container}>
+    <StaticGradientBackground />
+    <View style={styles.content}>
+      <Image
+        accessibilityIgnoresInvertColors
+        accessible={true}
+        resizeMode="contain"
+        style={styles.icon}
+      />
+      <H5 style={styles.text}>{I18n.t("features.itWallet.title")}</H5>
```

Comment:
As per Figma, the text should be a `Caption`
```suggestion
      <Caption style={styles.text}>{I18n.t("features.itWallet.title")}</Caption>
```